mobile/cafe.go

```diff
@@ -6,12 +6,12 @@ import (
 )
 
 // RegisterCafe calls core RegisterCafe
-func (m *Mobile) RegisterCafe(peerId string) error {
-	if !m.node.Online() {
-		return core.ErrOffline
+func (m *Mobile) RegisterCafe(host string) error {
+	if !m.node.Started() {
```

Comment:
Since we're not relying on the swarm, the node just needs to be started to register, not online.

Reply:
cool, will be faster to launch the app the first time then